README.md

```diff
@@ -2,7 +2,7 @@
 -------------------------------------------------------
 
 
-![Build status](https://travis-ci.org/ENCODE-DCC/encoded.svg?branch=master)
+[![CircleCI](https://circleci.com/gh/ENCODE-DCC/encoded/tree/dev.svg?style=svg)](https://circleci.com/gh/ENCODE-DCC/encoded/tree/dev)
```

Comment:
I wonder if there is a way to tie this badge to the current branch?  We don't really want to edit the README twice every time.

Reply:
You'd probably have to interpolate the branch into the readme

Reply:
we have a cloud config update coming that builds cloud configs using python string interpolation.  If we include the travis/circle config in that process then we can do stuff like this.